src/wexond/main/view-manager.ts

```diff
@@ -48,19 +50,14 @@ export class ViewManager {
       'browserview-destroy': (e: Electron.IpcMessageEvent, id: number) => {
         this.destroy(id)
       },
-      'browserview-call': async (e: any, data: any) => {
+      'browserview-call': async (e: any, data: { tabId: number, scope?: string, args: any[], callId?: string }) => {
         const view = this.views[data.tabId]
         if (!view) return
-        let scope: any = view
+        const scope: unknown = getValue(view, data.scope)
```

Comment:
If I'm not wrong the type is string no ? We can adapt the function getValue with generic return type 
```suggestion
        const scope = getValue<string>(view, data.scope)
```

Reply:
No it's not it may be a function, or anything else for that matter. Thus the `if (!scope || typeof scope !== 'function') return` below